plugin.xml

```diff
@@ -45,8 +44,15 @@
     <source-file src="src/ios/OSBarcodeError.swift" />
     <source-file src="src/ios/OSBarcodeScanArgumentsModel.swift" />
 
-    <source-file src="src/ios/frameworks/OSBARCArgumentMappable.swift" target-dir="extensions" />
-    <framework src="src/ios/frameworks/OSBarcodeLib.xcframework" embed="true" custom="true" />
+    <source-file src="src/ios/OSBARCArgumentMappable.swift" target-dir="extensions" />
 
+    <podspec>
+      <config>
+        <source url="https://cdn.cocoapods.org/"/>
+      </config>
+      <pods use-frameworks="true">
+        <pod name="OSBarcodeLib" spec="1.1.0" />
```

Comment:
Should we use something like `spec="~> 1"` so that the wrapper will use the latest version up till v2?

Reply:
You mean `spec="~> 1.1"`, right? 
We normally don't use the optimistic operator so we can avoid surprises. But in reality, we've just been mostly using 3rd party dependencies until now and this is entirely under our control, so it's different. 
How do you guys normally use it for the Capacitor plugins?

Reply:
Was having a look into Capacitor and noticed is using `spec="~> 1.1.0". Maybe we can use something like this here as well.

Reply:
I think for libs that we control then optimistic is a good idea, saves extra fuss when all we need is to update the native lib. The question for us is whether we want to go with `~> 1` which will give us all versions from `1.0.0` to but not including `2.0.0` or if we want to pin at minors with `~> 1.1` which gives us all version from `1.1.0` to but not including `1.2.0`

Reply:
Now that I'm thinking about this, on the low/no code side, I don't know if the optimistic operator is a good idea. To most OutSystems developers, the mobile app generated by MABS is a big black box. The same can be applied to the plugins. And since we have most of the code living on the native side, this would mean that most bug fixes would be fixed and non-breaking features would be done without any acknowledgement to the customer (for bug fixing i.e., they have Support Case and Problem tickets to track, that we reference when releasing a new version with the fix). 
Just discussed this with Luís Silva and he also believes that the optimistic operator could be troublesome.

Reply:
If its an issue for /low/no code customers then we can not use it that's fine